airflow/dags/cdc_restricted.py

```diff
@@ -91,10 +141,15 @@
 
 # CDC Restricted Data Ingestion DAG
 (
-    dummy_operator >> [
-        cdc_restricted_bq_op_race,
-        cdc_restricted_bq_op_sex,
-        cdc_restricted_bq_op_age
+    cdc_restricted_bq_op_sex_county >>
+    cdc_restricted_bq_op_age_county >> [
+        cdc_restricted_bq_op_race_state,
+        cdc_restricted_bq_op_sex_state,
+        cdc_restricted_bq_op_age_state
+    ] >> cdc_restricted_bq_op_race_county >> [
```

Comment:
Should this county task be grouped with the others? (not a suggestion, just curious)

Reply:
I think we can just run them all in parallel so I'll try that. I had moved it in the middle because you cant have two subsequent lists of tasks; so I was trying to break them up with the single longer task. Could also use a dummy operator there instead